src/argilla_sdk/settings/_resource.py

```diff
@@ -202,6 +204,38 @@ def serialize(self):
         except Exception as e:
             raise ArgillaSerializeError(f"Failed to serialize the settings. {e.__class__.__name__}") from e
 
+    def to_disk(self, path: str) -> None:
+        """Save the settings to a file on disk
+
+        Parameters:
+            path (str): The path to save the settings to
+        """
+
+        with open(path, "w") as file:
+            json.dump(self.serialize(), file)
```

Comment:
I realized that we're not serializing vectors and metadata properties. I will create a separate issue for this. This should be supported for 2.0 release

Reply:
Done https://github.com/argilla-io/argilla-python/issues/196